src/ucs/Makefile.am

```diff
@@ -43,6 +43,7 @@ nobase_dist_libucs_la_HEADERS = \
 	sys/string.h \
 	sys/sock.h \
 	sys/stubs.h \
+        type/spinlock.h \
```

Comment:
why added another time? it causes build failure

Reply:
I would like to fix this error
https://github.com/openucx/ucx/pull/3931#discussion_r305755908 without `#include <ucs/type/spinlock.h>` in `src/ucs/memory/rcache_int.h`
Could you tell me how to fix it if you know?

Reply:
I overlooked @dmitrygx comments.
https://github.com/openucx/ucx/pull/3931#discussion_r305784603
Revert this change.

It still requires `#include <ucs/type/spinlock.h>` statement in `src/ucs/memory/rcache_int.h
`.